src/clj_jargon/tickets.clj

```diff
@@ -140,3 +167,26 @@
   [cm user ticket-id]
   (proxy-input-stream cm (ticket-input-stream cm user ticket-id)))
 
+(defn iget
+  "Transfers remote-path to local-path using a ticket, using tcl as the TransferStatusCallbackListener"
+  ([cm ticket-id remote-path local-path tcl]
+    (iget cm ticket-id remote-path local-path tcl tcb))
+  ([cm ticket-id remote-path local-path tcl control-block]
+    (let [tco (ticket-client-operations cm)
```

Comment:
I don't feel like it makes sense to impersonate anyone here at present -- I could only really see that being useful if you're the admin, and if you're the admin it's probably easier to use a client username anyway? Plus, as admin why bother using the ticket-specific stuff, since you can just go for the underlying file. Basically, I don't think this needs a username parameter.